### ISSUE TITLE
Adopt new Communitybridge/Linux Foundation CLA bot

### ISSUE THREAD

**Issue by lizthegrey** [open-telemetry/community#309]
https://lfcla.com/ is the new Linux Foundation CLA bot, and we should switch to using it as it's less cumbersome than the existing Linux Foundation Account requirement.

**Comment by lizthegrey** [comment]
Yes, it's done. Most CLAs were migrated, but some may need to be re-signed.

**Comment by lizthegrey** [comment]
@flands For you, I see signatures migrated from Omnition but there is no signature on file for Splunk. Please follow https://docs.linuxfoundation.org/docs/communitybridge/communitybridge-easycla/ccla-managers-and-ccla-signatories/sign-in-to-the-cla-corporate-console to get Splunk as an authorized signatory.

**Comment by flands** [comment]
Thanks -- I will take a look

**Comment by lizthegrey** [comment]
See email just sent to otel maintainers: you'll see both checks in Github checks but you need to take action to migrate which check is `Required` in your branch protection settings.

**Comment by flands** [comment]
Got it -- concern is just hearing about this now (perhaps it was communicated somewhere else and missed it). Not sure how many contributors/maintainers are impacted, but if maintainers toggle this it seems like folks may be impacted. Trying to mitigate on the Splunk side now.

**Comment by lizthegrey** [comment]
> Got it -- concern is just hearing about this now (perhaps it was communicated somewhere else and missed it). Not sure how many contributors/maintainers are impacted, but if maintainers toggle this it seems like folks may be impacted. Trying to mitigate on the Splunk side now.

We believe a majority of contributors are covered under existing CLAs, we did an audit of the top 50 contributing organizations prior to this, and as I said, we've asked maintainers to slowly flip it according to their SIG's migration state with a 2-week rough time to migrate.

**Comment by trask** [comment]
> See email just sent to otel maintainers

How do I sign up for this email list? Just want to make sure I don't miss something important. Thanks!

**Comment by SergeyKanzhelev** [comment]
@trask let me look into it. 

@lizthegrey currently the mailing list does not contain maintainers. Perhaps you can post the mention and mention maintainers via teams (like @open-telemetry/java-instrumentation-approvers  )?

Also I'm not sure if maintainers with the "Maintain" permissions has access to `master` branch checks enforcing.

**Comment by lizthegrey** [comment]
oh, interesting, `cncf-opentelemetry-maintainers@lists.cncf.io` isn't the right way to email all maintainers? TIL.

well, let's do it this way then:

@open-telemetry/dotnet-maintainers 
@open-telemetry/php-maintainers 
@open-telemetry/javascript-maintainers 
@open-telemetry/java-instrumentation-maintainers 
@open-telemetry/java-maintainers 
@open-telemetry/go-maintainers (already done for y'all <3)
@open-telemetry/ruby-maintainers 
@open-telemetry/swift-mainteiners 
@open-telemetry/cpp-maintainers 
@open-telemetry/collector-maintainers 
@open-telemetry/erlang-maintainers 
@open-telemetry/python-maintainers 
@open-telemetry/opentelemetry-python-contrib-maintainers 
@open-telemetry/rust-maintainers 
@open-telemetry/docs-maintainers 
@open-telemetry/docs-cn-maintainers 

```
ACTION REQUIRED: once a majority of incoming PRs are passing the EasyCLA check, or two weeks from now, whichever is sooner, please disable the cla/linuxfoundation check in your repo settings under branch protection, and instead enable the EasyCLA check as the required check.

Details (copy-pasted from Gitter):

@/all we are implementing the new CNCF CLA bot, you may see the EasyCLA bot listed in status checks for your PRs on OTel repos. We are running both CLA bots in parallel for now, but it's up to your maintainers' discretion when they switch over, depending upon how long it takes for a majority of contributors to that SIG to sign in the new CLA system before it begins being enforced
it's the same CLA text, but in a new system and not every user necessarily has the correct association/has migrated over perfectly, thus the transition period.
more information & we welcome comments at https://github.com/open-telemetry/community/issues/309 -- notably, the new system more easily allows us to allowlist bots like Renovate and Dependabot, and also makes it easier for individuals to sign the CLA without creating Linux Foundation accounts.
oh, and I think it no longer requires bumping with dummy commits to re-run if you sign the CLA after getting told that it failed initial check
```

**Comment by tigrannajaryan** [comment]
> > See email just sent to otel maintainers
> 
> How do I sign up for this email list? Just want to make sure I don't miss something important. Thanks!

FWIW, I am a maintainer and I haven't seen any emails either. I cannot merge any PRs anymore and I suspect nobody from Splunk can contribute.

Would be good to fix the mailing list so that we are not caught by surprise next time changes like this happen.

**Comment by tigrannajaryan** [comment]
BTW, this is posted on Gitter but I am not sure I fully understand what it means:

>we are implementing the new CNCF CLA bot, you may see the EasyCLA bot listed in status checks for your PRs on OTel repos. We are running both CLA bots in parallel for now, but it's up to your maintainers' discretion when they switch over, depending upon how long it takes for a majority of contributors to that SIG to sign in the new CLA system before it begins being enforced

It would be nice if we actually had the grace which this appears to hint to.

**Comment by tigrannajaryan** [comment]
@open-telemetry/admins Is there a way to postpone the new CLA bot rollout for spec/proto repos for the 2 weeks that was mentioned as grace period to switch? It is going to take several days at least for my company to review the new CLA and in the meantime I have time-sensitive PRs to merge that I am unable to.

**Comment by tsloughter** [comment]
I also don't seem to  be on that mailing list. 

But also, I don't see "easycla" in the branch protections on the Erlang repos.

Is it supposed to be an action that runs? I don't see an entry for that on the repos either.

**Comment by tigrannajaryan** [comment]
> @open-telemetry/admins Is there a way to postpone the new CLA bot rollout for spec/proto repos for the 2 weeks that was mentioned as grace period to switch? It is going to take several days at least for my company to review the new CLA and in the meantime I have time-sensitive PRs to merge that I am unable to.

OK, it appears the issue not so urgent. The new EasyCLA bot shows that the build is failing, but actually it is misleading, it is still is possible to merge builds. I think we are good for now and have enough time to address this.

**Comment by lizthegrey** [comment]
> > @open-telemetry/admins Is there a way to postpone the new CLA bot rollout for spec/proto repos for the 2 weeks that was mentioned as grace period to switch? It is going to take several days at least for my company to review the new CLA and in the meantime I have time-sensitive PRs to merge that I am unable to.
> 
> OK, it appears the issue not so urgent. The new EasyCLA bot shows that the build is failing, but actually it is misleading, it is still is possible to merge builds. I think we are good for now and have enough time to address this.

Correct, it's an advisory but non-mandatory warning until the repo is switched over.
